docs/index.bs

```diff
@@ -2717,12 +2725,20 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       : Input
       :: |serviceWorker|, a [=/service worker=]
       :: *force bypass cache for importscripts flag*, an optional flag unset by default
-      :: optional |callbackSteps|, an algorithm which takes a [=Completion=]
 
+      <div class="note">
+        Note: This algorithm typically blocks until the service worker is [=running=]. Most callsites can check for success or failure by checking if the service worker is [=running=] after this algorithm returns.
+
+        This specification generally treats an [=abrupt completion=] due to an uncaught exception the same as a normal completion. That is, if the worker throws an exception during initial script evaluation, it is still considering running and can receive events. Some callsites distinguish between these scenarios by examining the [=start status=] of the [=running=] worker.
```

Comment:
Would it be cleaner to return *failure* or the start status from this algorithm?

That means callers could use

1. If the result of invoking [=Run Service Worker=] with |someServiceWorker| is *failure*, then return.

Or:

1. If the result of invoking [=Run Service Worker=] with |someServiceWorker| is *failure* or an [=abrupt completion=], then….

Reply:
Oh yes that would be much simpler... I'll do that.